osgp/shared/osgp-dto/src/main/java/org/opensmartgridplatform/dto/valueobjects/smartmetering/SetEncryptionKeyExchangeOnGMeterRequestDto.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2022 Alliander N.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.opensmartgridplatform.dto.valueobjects.smartmetering;
+
+public class SetEncryptionKeyExchangeOnGMeterRequestDto implements ActionRequestDto {
```

Comment:
Why not naming this newly introduced requestDto **SetKeyOnGMeterRequestDto** which is more appropriate as you gave a new name to the newly implemented command executor

Reply:
right, name of the requestDto is updated to SetKeyOnGMeterRequestDto, in line with the SetKeyOnGMeterCommandExecutor